multichain.go

```diff
@@ -284,16 +282,14 @@ func (asset Asset) ChainType() ChainType {
 // Type returns the asset-type (Native or Token) for the given asset.
 func (asset Asset) Type() AssetType {
```

Comment:
This function only works for Evm assets and erc20s, wonder if this is intentional?

Reply:
I think it's more that it's only necessary to differentiate for chains that support multiple kinds of assets.

Reply:
Got it.